velox/connectors/hive/HiveConfig.cpp

```diff
@@ -191,6 +191,21 @@ uint64_t HiveConfig::orcWriterMaxDictionaryMemory(const Config* session) const {
       core::CapacityUnit::BYTE);
 }
 
+bool HiveConfig::orcWriterLinearStripeSizeHeuristics(
+    const Config* session) const {
+  return session->get<bool>(
+      kOrcWriterLinearStripeSizeHeuristicsSession,
+      config_->get<bool>(kOrcWriterLinearStripeSizeHeuristics, true));
+}
+
+uint64_t HiveConfig::orcWriterMinCompressionSize(const Config* session) const {
+  return toCapacity(
+      session->get<std::string>(
+          kOrcWriterMinCompressionSizeSession,
+          config_->get<std::string>(kOrcWriterMinCompressionSize, "1kB")),
```

Comment:
nit: 1KB

Reply:
has to be kB for historical reasons. KB will not work.

Reply:
@tanjialiang, the semantics of this config is actually item count, not bytes, so let's stick to 1024. Also, to me it would be more flexible when serialized into a string map not to have special literals. WDYT?